sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/SqlSampleTests.java

```diff
@@ -69,6 +76,7 @@ public void testManageSqlWithRecoveredOrRestoredDatabase() {
     }
 
     @Test
```

Comment:
why disabling the those tests?

Reply:
A test exception like 
```
Header differences:
    <Accept> values differ, request <application/json>, record <application/json; q=0.9>
Body differences:
```
occurs. Since this release as  `Premiun Preview` version, the test cases where the exception occurs are disabled.